__tests__/components/HTMLBlock.test.jsx

```diff
@@ -0,0 +1,33 @@
+const { mount } = require('enzyme');
+const React = require('react');
+const { renderToString } = require('react-dom/server');
+
+const sanitize = require('../../sanitize.schema');
+const HTMLBlock = require('../../components/HTMLBlock')(sanitize);
+
+describe('HTML Block', () => {
+  global.window = true;
+  global.mockFn = jest.fn(() => console.log('custom script ran'));
```

Comment:
This looks like it could cause test leakage. Switching the order of the tests causes the `.toHaveBeenCalledTimes(0)` to fail.

I think these ought to be in a `beforeEach` block.

Reply:
Good call; refactored in #103.